Add MFT to geo file for vertexing

njacazio opened the pull request.

njacazio commented:
@mmazzilli this should fix the issue with the log, could you try now?

mmazzilli commented:
> @mmazzilli this should fix the issue with the log, could you try now?

Thanks a lot @njacazio. I tested it and indeed it fixed the previous problem, but now a segmentation violation occurs when the MID is configured/created in the simulation.

njacazio commented:
> > @mmazzilli this should fix the issue with the log, could you try now?
> 
> Thanks a lot @njacazio. I tested it and indeed it fixed the previous problem, but now a segmentation violation occurs when the MID is configured/created in the simulation.

Hi, it seems unrelated to the MID. I added checks about errors and fixed the issue you were observing in here  https://github.com/AliceO2Group/DelphesO2/pull/85 Thanks for spotting this!
